src/rt-app.c

```diff
@@ -495,18 +525,18 @@ static int run_event(event_data_t *event, int dry_run,
 		break;
 	case rtapp_suspend:
 		{
-		log_debug("suspend %s ", rdata->name);
-		pthread_mutex_lock(&(ddata->res.mtx.obj));
-		pthread_cond_wait(&(rdata->res.cond.obj), &(ddata->res.mtx.obj));
-		pthread_mutex_unlock(&(ddata->res.mtx.obj));
+			log_debug("suspend %s ", rdata->name);
```

Comment:
This and right below seems unrelated to this change?

Reply:
the 1st commit is doing a cleanup 
the 2nd fix the unique timer problem